x-pack/plugins/fleet/server/services/setup.ts

```diff
@@ -242,28 +251,3 @@ export async function setupFleet(
 function generateRandomPassword() {
   return Buffer.from(uuid.v4()).toString('base64');
 }
-
-async function addPackageToAgentPolicy(
```

Comment:
I would like to keep this method here instead of moving it to the preconfiguration service since it is generic, or maybe we can consider moving it to `services/agent_policy.ts`